server/svix-server/src/v1/endpoints/endpoint.rs

```diff
@@ -449,11 +452,7 @@ async fn rotate_endpoint_secret(
 
         old_keys: Set(Some(ExpiringSigningKeys(
             iter::once(last_key)
-                .chain(
-                    old_keys
-                        .map(|x| x.0.into_iter())
-                        .unwrap_or_else(|| vec![].into_iter()),
-                )
+                .chain(old_keys.map_or_else(|| vec![].into_iter(), |x| x.0.into_iter()))
```

Comment:
Side comment: silly decision on Rust to have the map the SECOND parameter and the else the first. It's literally called "map or else", in this order...